src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_client_factory.py

```diff
@@ -43,4 +43,7 @@ def cf_acr_webhooks(cli_ctx, *_):
 
 
 def cf_acr_builds(cli_ctx, *_):
-    return get_acr_build_client(cli_ctx).builds
\ No newline at end of file
+    return get_acr_build_client(cli_ctx).builds
+
```

Comment:
Two empty lines between methods. Or run pylint to fix all such issues ...

Reply:
Resolved